mysite/announcements/urls.py

```diff
@@ -4,4 +4,5 @@
 urlpatterns = [
     url(r'^$', views.home, name='home'),
     url(r'^register/', views.register, name='register'),
+    url(r'^api/(?P<pk>[0-9]+)/$', views.ListStatusView.as_view())
```

Comment:
`pk` of what?

`api` of what? What does this do?

Reply:
pk for entries in Status table
api to get endpoints of Status table

Reply:
So shouldn't this be the explicit? In naming